### PR TITLE
Updated 'contact us' page and contact info in header and footer sections across all pages.

### PR DESCRIPTION
**Pull request by Nancique**

### CODE REVIEW

File: about_us.html [hunk 1]
```diff
@@ -269,14 +269,12 @@ <h2>Useful Links</h2>
...
-              Phone: +91 9159669599 <br />
-              Email:
-              <a href="mailto:info@anybiz.com" class="">info@bluedart.in</a
-              ><br />
-              Web: <a href="smart-eye.html" class="">www.bluedart.in</a>
+              7, Omo Ighodalo Street, <br>
+              Ogudu GRA,<br>
+              Ogudu, Lagos.<br>
+              Phone: (+234) 904-329-8729<br>
+              Email: <a href="mailto:info@anybiz.com" class="">support@techknights.com</a><br>
```
Comment (Didimukhtar) [inline about_us.html]: Replace all Href with a Hashtag ( # )

File: services.html [hunk 2]
```diff
@@ -213,12 +213,12 @@ <h2>Useful Links</h2>
...
-                        Tamil Nadu, IND <br>
-                        Phone: +91 9159669599 <br>
-                        Email: <a href="mailto:info@anybiz.com" class="">info@bluedart.in</a><br>
-                        Web: <a href="smart-eye.html" class="">www.bluedart.in</a>
+                        7, Omo Ighodalo Street, <br>
+                        Ogudu GRA,<br>
+                        Ogudu, Lagos.<br>
+                        Phone: (+234) 904-329-8729<br>
+                        Email: <a href="mailto:info@anybiz.com" class="">support@techknights.com</a><br>
+                        Web: <a href="smart-eye.html" class="">www.notohungers.com</a>
```
Comment (Didimukhtar) [inline services.html]: Spelling error; Remove "s"

**Review by Didimukhtar — CHANGES_REQUESTED** [review]
Replace all Href link with ( # ), Remove the "s" in notohunger.com located at the footer of services.html

**Review by Didimukhtar — APPROVED** [review]

File: about_us.html (hunk 1 above)
Comment (Nancique) [reply about_us.html]: Alright I'll edit those.

**Comment by Nancique** [comment]
Done with the edits. Do I create another pull request?

**Review by Didimukhtar — APPROVED** [review]
Changes approved

**Pull request merged**